<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VhallLiveTitle</class>
 <widget class="QWidget" name="VhallLiveTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VhallLiveTitle</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_logo">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CLabel" name="label_RoomTitle">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>互联网大会</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="webinarIdWdg" name="widget_room_id" native="true">
        <property name="minimumSize">
         <size>
          <width>126</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>126</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget_room_id{
border-radius:2px;
border:1px solid rgba(151,151,151,1);
}
</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_image"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CLabel" name="label_userName">
        <property name="styleSheet">
         <string notr="true">
color: rgb(204, 204, 204);</string>
        </property>
        <property name="text">
         <string>微吼直播</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonWdg" name="RecordBtnWdg" native="true">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_toSpeak">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image: url(:/toolWidget/img/toolWidget/toSpeak.png) 0 182 0 0;
	color: rgb(255, 249, 249);
}
QPushButton:hover{
	border-image: url(:/toolWidget/img/toolWidget/toSpeak.png) 0 91 0 91;
	color: rgb(255, 253, 253);
}

QPushButton:pressed{
	border-image: url(:/toolWidget/img/toolWidget/toSpeak.png) 0 182 0 0;
	color: rgb(255, 253, 253);
}

QPushButton:disabled{		
   border-image: url(:/toolWidget/img/toolWidget/toSpeak.png)0 182 0 0;
	color: rgb(255, 216, 216);
}

</string>
        </property>
        <property name="text">
         <string>申请上麦</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CTimeButton" name="pushButton_startLive">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PushButton" name="pushButton_setting">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="pushButton_fresh">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="pushButton_minSize">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="pushButton_fullSize">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_close">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border-image: url(:/sysButton/img/sysButton/close_btn.png);
}
QPushButton:hover{
border-image: url(:/sysButton/img/sysButton/close_btn_hover.png);
}

QToolTip{ border:1px solid rgb(0, 0, 0, 1); background-color: rgba(0, 0, 0, 178); color:#ffffff; font-size:12px; }

</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>push_button.h</header>
  </customwidget>
  <customwidget>
   <class>ButtonWdg</class>
   <extends>QWidget</extends>
   <header>buttonwdg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CLabel</class>
   <extends>QLabel</extends>
   <header>clabel.h</header>
  </customwidget>
  <customwidget>
   <class>webinarIdWdg</class>
   <extends>QWidget</extends>
   <header>webinaridwdg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CTimeButton</class>
   <extends>QPushButton</extends>
   <header location="global">ctimebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../VhallLive/project/VhallLive.qrc"/>
 </resources>
 <connections/>
</ui>
